File: app/src/main/java/com/hubwallet/apptspos/Utils/Models/CustomerModel/CustomerData.java
package com.hubwallet.apptspos.Utils.Models.CustomerModel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class CustomerData {

    @SerializedName("customer_data")
    @Expose
    private List<CustomerDatum> customerData = null;

    public List<CustomerDatum> getCustomerData() {
        return customerData;
    }

    public void setCustomerData(List<CustomerDatum> customerData) {
        this.customerData = customerData;
    }

}

File: app/src/main/java/com/hubwallet/apptspos/retrofit/login/CustomerData.kt
package com.hubwallet.apptspos.retrofit.login


import com.google.gson.annotations.SerializedName
import androidx.annotation.Keep

@Keep
data class CustomerData(
    @SerializedName("customer_id")
    var customerId: String = "",
    @SerializedName("customer_name")
    var customerName: String = "",
    @SerializedName("email")
    var email: String = ""
)